Xplat/src/main/java/vazkii/botania/data/LooniumStructureLootProvider.java

```diff
@@ -179,7 +179,7 @@ public CompletableFuture<?> run(@NotNull CachedOutput cache) {
 		for (var e : tables.entrySet()) {
 			Path path = pathProvider.json(e.getKey());
 			LootTable.Builder builder = e.getValue();
-			LootTable lootTable = builder.setParamSet(LootContextParamSets.EMPTY).build();
+			LootTable lootTable = builder.setParamSet(LootContextParamSets.ALL_PARAMS).build();
```

Comment:
Shouldn't this ideally be changed to the context parameters we actually supply/use?

Reply:
Could maybe be solved with a special loot function that makes the table ignore errors/warnings for missing parameters in a smooth way, that can be applied if the source loot table expects some parameters. But specifying all_params is probably fine for now